demos/drawer/drawer.scss

```diff
@@ -14,7 +14,24 @@
 // limitations under the License.
 //
 
+@import "@material/list/mixins";
+@import "@material/form-field/mdc-form-field";
```

Comment:
For demo pages, we use relative paths in our Sass `@import`s (demo pages don't need to be distributed via npm).